assets/javascripts/modules/checkout/voucherFields.jsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import moment from 'moment'
+import formElements from './formElements'
+import CustomDatePicker from '../react/customDatePicker'
+import CharacterCountedTextArea from './characterCountedTextArea'
+import reviewDetails from './reviewDetails'
+
+require('react-datepicker/dist/react-datepicker.css');
+
+const MAX_WEEKS_AVAILABLE = 4;
+
+function filterDate() {
+    return (date) => date.day() === 1;
```

Comment:
This is fine for a first pass, but we should consider making it show the relevant day to the chosen package. e.g. Everyday and Sixday shows Monday. Weekend shows Saturday and Sunday+ shows Sunday.